ophyd/areadetector/filestore_mixins.py

```diff
@@ -63,8 +64,10 @@ class FileStoreBase(BlueskyInterface, GenerateDatumInterface):
     `datetime.strftime`, which accepts the standard tokens, such as
     %Y-%m-%d.
     """
-    def __init__(self, *args, fs=None, write_path_template=None,
-                 read_path_template=None, **kwargs):
+    def __init__(self, *args, fs=None,
```

Comment:
Random question: when you touch function signatures, they usually end up with 2-3 args per line instead of the usual however-many-fit-in-79-characters. Just a quirk or an emacs thing?